### ISSUE TITLE
Add public location data

### ISSUE THREAD

**Issue by nwebb-roux** [cs7290/stinky#9]

**Comment by nwebb-roux** [comment]
https://observablehq.com/d/f305f4a853afe835 - all Maine schools

**Comment by nwebb-roux** [comment]
@SilentMutant is working on this, along with tooltips to show location info on mouseover. Planning to team up on debugging during the coming week.

**Comment by nwebb-roux** [comment]
https://observablehq.com/d/5849f4854a46ae00

**Comment by bergsalex** [comment]
Those tooltips have been the death of me whenever I tried to implement them in observable.

**Comment by SilentMutant** [comment]
https://observablehq.com/d/3bfa740e34943fbb
Notebook with currently working popups.

**Comment by PhilipMathieu** [comment]
Looks good. I was able to pull it into the main notebook: https://observablehq.com/compare/e5057433bd482160@1118...e5057433bd482160@1130

Are we waiting to do anything with the styling?

**Comment by SilentMutant** [comment]
I was going to upload some of symbols to imgur so we use those instead of dots.

**Comment by SilentMutant** [comment]
Also trying to get more geojson data other than schools.

**Comment by SilentMutant** [comment]
Used Maptiler cloud to create cusom datasets for child care and parks. These can be edited so if you see I missed something in my data set let me know and I can add it. Govt data will be redone using this as the one I found online seems to be bad. Temp styling new data as circles in different colors to ensure its functioning before I replace them all with different colored symbols.

**Comment by PhilipMathieu** [comment]
Cool that you're using MapTiler - I remember encountering it for a previous project but not having enough reason to learn how to use it.

**Comment by nwebb-roux** [comment]
Hey all, what's going on with the versioning in the Observable notebook? It looks like @1130 has a bunch of features that are missing from the latest version, @1171. Am I missing something?

**Comment by PhilipMathieu** [comment]
> Hey all, what's going on with the versioning in the Observable notebook? It looks like @1130 has a bunch of features that are missing from the latest version, @1171. Am I missing something?

I was trying to incorporate Silent Mutant's work but then he clarified that it wasn't quite ready at the time so I reverted. This is definitely proving to be a weak point of the Observable collaboration system because unless you're actually editing together in real-time it's very hard to follow what other people have done in the interim.



> I have been working in the original notebook we got popups working in to avoid messing with the one that is linked to the site directly. I have fixed all the data and symbols and it should be good to just copy over the code into the live notebook if everyone agrees.

I think it looks good-to-go. Let me know if you need any help - I'm working on final project stuff for other classes all day and can jump on whenever.

**Comment by nwebb-roux** [comment]
@SilentMutant Yeah man, I think you should go ahead and bring it all into the live notebook. Then I can use that as a jumping off point to get the Fly To running.

**Comment by bergsalex** [comment]
@PhilipMathieu I may have undone your reversion. I added the schools with school like icons and the fly to. 

@SilentMutant you'll need to take it from where I left off in the main notebook. I have schools only for icons and fly to at the moment. I used the example code from you and @nwebb-roux.

**Comment by nwebb-roux** [comment]
Fly To and icons look good for schools, nice work @bergsalex! We just need to integrate the rest of the locations.

**Comment by SilentMutant** [comment]
Sorry for the late response I was at work all day. I just integrated it all with the new changed symbols.
